TestCntlrApp/src/enbApp/nb_dam.c

```diff
@@ -911,26 +908,21 @@ PUBLIC S16 nbDamPcapDatInd(Buffer *mBuf)
   }
 
   /* Fetch ToS or DSCP*/
-  for (idx = 0; idx < 1; idx++) {
-    if ((SExamMsg(&ipPkt[idx], mBuf, ipIdx) != ROK)) {
-      NB_LOG_ERROR(&nbCb, "Failed to fetch ToS");
-      SPutMsg(mBuf);
-      RETVALUE(RFAILED);
-    }
-    ipIdx++;
+  U8 idx = 0;
+  if ((SExamMsg(&ipPkt[idx], mBuf, ipIdx) != ROK)) {
```

Comment:
why not directly write `&ipPkt[0]` instead of `&ipPkt[idx]`

Reply:
done